.github/workflows/backend-ci-cd.yml

```
@@ -0,0 +1,99 @@
+          for i in {1..30}; do
+            if docker compose -f docker-compose.test.yaml ps | grep "healthy"; then
+              echo "Infrastructures are healthy"
+              break
+            fi  
+            echo "Waiting for infrastructures to be healthy...($i/30)"
+            sleep 10
+          done
+          docker compose -f docker-compose.test.yaml ps | grep "healthy" || exit 1
+      - name: Run Integration Tests for ${{ matrix.service }}
+        working-directory: apps/backend/${{ matrix.service }}
+        env:
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)